.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## TL;DR
+
+<!-- Put 1-2 sentences here, at max. -->
+
+## Depends on
+
+<!-- add a link here for something this PR depends on, like another PR -->
+
+## Issues
+
+<!-- Paste any issue links here, even issues from Gitlab --> 
+
+## Changelog
```

Comment:
nit: How about changing this to "Proposed Changes" or something?
Since `changie` manages our "Changelog" this feels a bit overloaded

Reply:
Agree

Reply:
@davidbloss 

> Since changie manages our "Changelog" this feels a bit overloaded

Great point. Also the intention behind the author's PR matters a lot more when you're reading the description. I'm updating this section